### ISSUE TITLE
add stdin passing to container runtime in E wrapper

### ISSUE THREAD

**Issue by mkyc** [epiphany-platform/epiphany#1499]
similar to `Edit` section [here](https://stackoverflow.com/q/58732588)

**Comment by rpudlowski93** [comment]
@mkyc As I understand, We would like to use these commands: [here](https://github.com/mkyc/epiphany-wrapper-poc/releases/tag/0.0.1 ) and additionally we would like to be able pass stdin there? Could You provide here any example in order to explain usages?

**Comment by mkyc** [comment]
Yeah. So for example if you run `terraform apply` you can pass `-auto-approve` parameter and then you **do not need** stdin. But if you do not want to auto approve this, you would need to connect to container stdin and pass your keyboard bytes to it.
